tests/unit/plugins/lookup/test_dsv.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# (c) 2020, Adam Migus <adam@migus.org>
+#
+# This file is part of Ansible
```

Comment:
Please use the short GPL header for this (`GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)` in one line below the copyright).